app/assess/models/application.py

```diff
@@ -49,3 +49,11 @@ def get_question(self, index: int):
         if self.questions:
             return self.questions[index]
         return None
+
+    def get_question_field(self, page_title):
```

Comment:
This should maybe be get_question_field**s** plural?

Reply:
changed :)